crates/sargon/src/profile/v100/networks/network/profile_network.rs

```diff
@@ -174,6 +174,32 @@ impl ProfileNetwork {
             None
         }
     }
+
+    /// Hides the account associated with the `account_address`
+    pub(crate) fn hide_account(
+        &mut self,
+        account_address: &AccountAddress,
+    ) -> Option<Account> {
+        let account = self.update_account(account_address, |account| {
+            account.mark_as_hidden();
+        });
+        self.authorized_dapps
+            .remove_referenced_account(account_address);
+        account
+    }
+
+    /// Tombostones the account associated with the `account_address`
```

Comment:
```suggestion
    /// Tombstones the account associated with the `account_address`
```

applies in other places as well

Reply:
strange that `typos` step in CI did not catch this.